src/controllers/managing.officer.controller.ts

```diff
@@ -1,10 +1,34 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
 import { logger } from "../utils/logger";
 import * as config from "../config";
+import { ApplicationData, ApplicationDataType, managingOfficerType } from "../model";
+import { getApplicationData, prepareData, setApplicationData } from "../utils/application.data";
 
 export const get = (req: Request, res: Response) => {
-  logger.info(`GET MANAGING_OFFICER_PAGE`);
+  logger.debug(`GET ${config.MANAGING_OFFICER_PAGE}`);
```

Comment:
should have a try -catch around the get function (like post)

Reply:
and a test that it handles teh error (see other controller tests)

Reply:
Agree. Weirdly, test is there and passing so will have a look.